src/CoreWCF.ConfigurationManager/src/CoreWCF/Configuration/AddressHeaderCollectionElement.cs

```diff
@@ -0,0 +1,81 @@
+﻿// Licensed to the .NET Foundation under one or more agreements.
+// The .NET Foundation licenses this file to you under the MIT license.
+
+using System.Configuration;
+using System.Security;
+using System.Xml;
+using CoreWCF.Channels;
+
+namespace CoreWCF.Configuration
+{
+    public sealed class AddressHeaderCollectionElement : ServiceModelConfigurationElement
+    {
+        public AddressHeaderCollectionElement()
+        {
+        }
+
+        internal void Copy(AddressHeaderCollectionElement source)
+        {
+            if (source == null)
+            {
+                throw DiagnosticUtility.ExceptionUtility.ThrowHelperArgumentNull("source");
+            }
```

Comment:
Can you replace string literals of method parameters with nameof. e.g. `nameof(source)`. I use the VS find and replace tool and turn on regex searching. You can set it to search all files in the project. The search string I use is `ThrowHelperArgumentNull\(\"([a-zA-Z0-9]+)\"\)` and in the replace textbox I use `ThrowHelperArgumentNull(nameof($1))`. There are sometimes the occasional place where the string used in the exception doesn't match the parameter name, but that's rare.